app/screens/MapScreen.js

```diff
@@ -137,41 +143,50 @@ class MapScreen extends React.Component {
 		}
 	}
 
-	input() {
-		this.setState({ label: 'hello world' })
+	onMapPress(data) {
+		const {
+			positionX,
+			positionY,
+			scale,
+			type,
+			zoomCurrentDistance
+		} = data;
+
+		// Only update when map stops moving
+		if (type !== "onPanResponderRelease") return;
+
+		// Placeholder for now
+		return { positionX, positionY, scale, type, zoomCurrentDistance };
```

Comment:
What is this supposed to return then?

Reply:
The function shouldn't return anything.  I created the other return so `npm run lint` wouldn't throw an error.

Reply:
I see, makes sense.